tools/cldr-apps/js/src/views/AboutPanel.vue

```diff
@@ -23,12 +23,10 @@
               <a class="aboutValue" v-bind:href="v">{{ v }}</a>
             </span>
             <span v-else-if="valueIsHash(v, k)">
-              <!-- Looks like a Git hash, 6…40 hex chars.
-                   We need to check that CLDR_COMMIT_BASE is set, because that’s the
-                   base URL for the linkification -->
+              <!-- Looks like a Git hash, 6…40 hex chars. -->
               <a
                 class="aboutValue"
-                v-bind:href="aboutData.CLDR_COMMIT_BASE + v"
+                v-bind:href="gitUrlPrefix + v + gitUrlSuffix"
```

Comment:
I actually don't like this change, because it's only correct in some situations, as I noted on the original ticket. For example, if an alternate branch or back level version is deployed, the "..main" diff may not return much.

What I would propose (and what I proposed there) is to have at least two links:
- a link directly to the commit (under the commit hash)
- a link with the comparison to "..main" (as 'vs. main' or some such)

I use this link often, and a link directly to the commit is what is most useful.

Reply:
good idea -- it's trivial to include both links in the about panel

Reply:
in https://unicode-org.atlassian.net/browse/CLDR-13843?focusedCommentId=158448 @macchiati  also asked for the 'changes since last released version'.  Not sure how you'd calculate that, but could use the OLDVERSION field from cldr.properties.

Reply:
`gitUrlPrefix + 'release-' + '41'  + '...' + v` would give all changes since v41  [example](https://github.com/unicode-org/cldr/compare/release-41...91c8dcdcce09065085c23a471ed1736f00aded35)

Reply:
OLDVERSION with `.` changed to `-`